mon-pix/app/components/routes/register-form.js

```diff
@@ -280,12 +277,16 @@ export default class RegisterForm extends Component {
 
   _updateInputsStatus() {
     const errors = this.schoolingRegistrationDependentUser.errors;
-    errors.forEach(({ attribute, message }) => {
-      const statusObject = 'validation.' + attribute + '.status';
-      const messageObject = 'validation.' + attribute + '.message';
-      this.set(statusObject, 'error');
-      this.set(messageObject, message);
-    });
+
+    if (errors) {
+      errors.forEach(({ attribute, message }) => {
+        const statusObject = 'validation.' + attribute + '.status';
+        const messageObject = 'validation.' + attribute + '.message';
+        this.set(statusObject, 'error');
+        this.set(messageObject, message);
+      });
+    }
+    this.set('registerErrorMessage', true);
```

Comment:
**Question** : On set `registerErrorMessage` à `true` même quand il n'y a rien dans `errors` ?
J'ai du mal à comprendre l'utilité de ce booléen.

Reply:
Sachant que le username n'est plus un input mais une span, nous nous sommes demandés comment afficher une erreur sur le formulaire dans le cas ou la valeur username était modifiée par l'utilisateur.

On a eu l'idée un booléen qui passerait à true si une erreur est remontée, mais ne faisant pas partie de `schoolingRegistrationDependentUser.errors`. 
Le message d'erreur, bien que pensé pour le cas username est assez général "Une erreur est survenue. Veuillez recommencer ou contacter le support." 

J'espère avoir éclairci cette étape. :)

Reply:
C'est clair ! Mais du coup la ligne `this.set('registerErrorMessage', true);` devrait être dans le if une ligne plus tôt non ?

Reply:
Il semble manquer un else.

Reply:
J'ai ajouté le else

Reply:
Ah oui je viens de comprendre l'intention avec ton message + le else.
Merci !